metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/ObjectsImporter.java

```diff
@@ -159,6 +199,41 @@ public Analysis importAnalysis(String accession) {
         return analysis;
     }
 
+    private ReferenceSequence buildReferenceSequenceOfSEQUENCEVARIATION(AnalysisType analysisType) {
+        ReferenceSequence referenceSequence = null;
+
+        AnalysisType.ANALYSISTYPE analysistype = analysisType.getANALYSISTYPE();
+        if (analysistype.isSetSEQUENCEVARIATION()) {
+            ReferenceSequenceType referenceSequenceType = analysistype.getSEQUENCEVARIATION();
+            if (referenceSequenceType != null) {
+                ReferenceAssemblyType referenceAssemblyType = referenceSequenceType.getASSEMBLY();
+                if (referenceAssemblyType != null) {
+                    ReferenceAssemblyType.STANDARD standard = referenceAssemblyType.getSTANDARD();
+                    if (standard != null) {
+                        String refName = standard.getRefname();
+                        String accession = null;
+                        if (refName != null) {
+                            accession = REFERENCE_MAP.get(refName.toLowerCase());
```

Comment:
this manual creation is just for database case / ega, so it should be moved to SraDatabaseImporter class

Reply:
Fixed by @iyangar, now moved to `SraObjectsImporterThroughDatabase`